bom/pom.xml

```diff
@@ -75,9 +76,12 @@
     <spring-security-crypto.version>5.2.4.RELEASE</spring-security-crypto.version>
     <truth.version>0.28</truth.version>
     <vertx.version>3.9.1</vertx.version>
-    <wildfly-common.version>1.5.1.Final</wildfly-common.version>
+    <wildfly-common.version>1.5.4.Final-format-001</wildfly-common.version>
     <wildfly-elytron.version>1.10.4.Final</wildfly-elytron.version>
     <wiremock.version>2.24.1</wiremock.version>
+    <quarkus.platform.version>1.6.1.Final</quarkus.platform.version>
+    <quarkus-plugin.version>1.6.1.Final</quarkus-plugin.version>
+    <jandex-plugin.version>1.0.7</jandex-plugin.version>
```

Comment:
FMPOV we should move the definition of the quarkus and Jandex plugins to the parent POM where we also define the other commonly used plugins. WDYT?

Reply:
I did that for now. My only concern is that it might slow down the build. I'll keep an eye on that and make additional fixes if needed.